audbackend/core/artifactory.py

```diff
@@ -292,12 +305,9 @@ def _ls(
             paths = [str(self._path(path, v))
                      for v in vs if self._exists(path, v)]
 
-            if not paths:
-                utils.raise_file_not_found_error(path)
```

Comment:
No longer needed, since a `FileNotFoundError` will be raised in line 305 in that case.